tools/unit-tests-merge.cs

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env /Library/Frameworks/Mono.framework/Commands/csharp
```

Comment:
we normally use `.csharp` for script files, so they are not confused with, to be compiled, `.cs` files